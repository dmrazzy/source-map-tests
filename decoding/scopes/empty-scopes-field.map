{
  "version": 3,
  "file": "empty-scopes-field.js",
  "sources": [],
  "mappings": "",
  "names": [],
  "scopes": ""
}