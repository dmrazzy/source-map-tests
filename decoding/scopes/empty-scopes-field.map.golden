{
  "file": "empty-scopes-field.js",
  "mappings": [],
  "sources": [],
  "ranges": []
}